game/templates/game/board.html

```
@@ -622,7 +658,21 @@ <h1>Checkora <span class="badge" id="modeBadge">PVP</span></h1>
-                <button class="btn" id="drawDeclineBtn" style="width: 90px; background: #333; color: #fff;">Decline</button>
+                <button class="btn" id="drawDeclineBtn"
+                    style="width: 90px; background: #333; color: #fff;">Decline</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- ========== FEN MODAL ========== -->
+    <div class="promo-overlay" id="fenOverlay">
+        <div class="promo-dialog">
+            <div class="promo-title">Start from FEN</div>
+            <input type="text" id="fenInput" class="fen-input" placeholder="Paste FEN string">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)